pkg/kubelet/kubelet_pods.go

```diff
@@ -487,6 +493,22 @@ func managedHostsFileContent(hostIPs []string, hostName, hostDomainName string,
 	return buffer.Bytes()
 }
 
+// isSupportIPv6 Determine whether the Pod supports ipv6 by checking
+// loopback interface inside the pod whether actually assigned IPv6 IP.
```

Comment:
this is not accurate, right?
This will look at the interface in the host, not in the pod

Reply:
Yes, this was a wrong description, fixed